connect/runtime/src/test/java/org/apache/kafka/connect/util/KafkaBasedLogTest.java

```diff
@@ -147,10 +168,6 @@ public void setUp() {
 
     @Test
     public void testStartStop() throws Exception {
-        expectStart();
-        expectStop();
```

Comment:
Don't we also lose coverage here? The current tests ensure that the producer is closed; we should retain those guarantees.